examples/sn-dms-demo/src/store/wopi/reducers.ts

```diff
@@ -0,0 +1,22 @@
+import { ODataWopiResponse } from '@sensenet/client-core'
+import { AnyAction, combineReducers, Reducer } from 'redux'
+
+const wopiDataInitialState = {
+  accesstoken: '',
+  actionUrl: '',
+  expiration: 0,
+  faviconUrl: '',
+}
+
+export const wopiData: Reducer<ODataWopiResponse> = (state = wopiDataInitialState, action: AnyAction) => {
+  switch (action.type) {
+    case 'GET_WOPIDATA_SUCCESS':
+      return action.result
+    default:
+      return state
+  }
+}
+
+export const wopi = combineReducers({
```

Comment:
Is it necessary to call combineReducers with one reducer?

Reply:
👍